strawberry_django_plus/type.py

```diff
@@ -89,13 +118,21 @@ def _from_django_type(
 
         field = cast(StrawberryDjangoField, field)
 
-        # FIXME: Improve this...
         if not field.base_resolver:
 
-            def conn_resolver(root):
-                return getattr(root, name).all()
+            def conn_resolver(parent_annotation: Optional[StrawberryAnnotation], root, info: Info):
```

Comment:
Also wondering if you really need to create the partial here.

In theory you could use `type_annotation` directly inside this function, since if is part of the locals while being defined